libs_changes.md

```diff
@@ -0,0 +1,115 @@
+amongst themselves.
+
+* If there are *tons* of RFCs -- especially trivial ones -- people are less
+likely to engage with them. Official team members are super busy. Domain experts
+and industry professionals are super busy *and* have no responsibility to engage
+in RFCs. Since these are *exactly* the most important people to get involved in
+the RFC process, it is important that we be maximally friendly towards their
+needs.
+
+
+# Is an RFC required?
+
+The overarching philosophy is: *do whatever is easiest*. If an RFC
+would be less work than an implementation, that's a good sign that an RFC is
+necessary. That said, if you anticipate controversy, you might want to short-circuit
+straight to an RFC. For instance new APIs almost certainly merit an RFC. Especially
+as `std` has become more conservative in favour of the much more agile cargoverse.
+
+* **Submit a PR** if the change is a:
+    * Bugfix
+    * Docfix
+    * Obvious API hole patch, such as adding an API from one type to a symmetric type.
+      e.g. `Vec<T> -> Box<[T]>` clearly motivates adding `String -> Box<str>`
+    * Minor tweak to an unstable API (renaming, generalizing)
+    * Implementing an "obvious" trait like Clone/Debug/etc
+* **Submit an RFC** if the change is a:
+    * New API
+    * Semantic Change to a stable API
+    * Generalization of a stable API (e.g. how we added Pattern or Borrow)
+    * Deprecation of a stable API
+    * Nontrivial trait impl (because all trait impls are insta-stable)
+* **Do the easier thing** if uncertain. (choosing a path is not final)
+
+
+# Non-RFC process
+
+* A (non-RFC) PR is likely to be **closed** if clearly not acceptable:
+    * Disproportionate breaking change (small inference breakage may be acceptable)
+    * Unsound
+    * Doesn't fit our general design philosophy around the problem
+    * Better as a crate
+    * Too marginal for std
+    * Significant implementation problems
+
+* A PR may also be closed because an RFC is approriate.
+
+* A (non-RFC) PR  may be **merged as unstable**. In this case, the feature
+should have a fresh feature gate and an associated tracking issue for
+stabilisation. Note that trait impls and docs are insta-stable and thus have no
+tracking issue. This may imply requiring a higher level of scrutiny for such
+changes.
+
+However, an accepted RFC is not a rubber-stamp for merging an implementation PR.
+Nor must an implementation PR perfectly match the RFC text. Implementation details
+may merit deviations, though obviously they should be justified. The RFC may be
+amended if deviations are substantial, but are not generally necessary. RFCs should
+favour immutability. The RFC + Issue + PR should form a total explanation of the
+current implementation.
+
+* Once something has been merged as unstable, a shepherd should be assigned
+  to promote and obtain feedback on the design.
+
+* Once the API has been unstable for at least one full cycle (6 weeks),
```

Comment:
@Gankro 

> Sounds good to me -- though meeting deadlines is not our forte, historically speaking.

I'm not sure what you have in mind here, but in any case I think the change I'm proposing will make it a lot easier to track/manage these details:

- Week before release: Go/no-go meeting for items under FCP
- Week of release: :confetti_ball: 
- Week after release: Triage for next cycle's stabilization FCP

@alexcrichton Note that I said "some number" not a "small number" ;-)

It's true that we have a backlog at the moment, and I think it's probably fair to grandfather in our existing unstable APIs which have been shipping since 1.0. Still, the main point here is that there's no reason for a stabilization FCP to be a single-week thing; it may as well last until the end of the cycle.

(On a separate note: I'd like to make progress this week moving forward on our remaining unstable APIs, and as part of that perhaps we can propose a bulk stabilization for this cycle.)